infra/Pos/Communication/Relay/Class.hs

```diff
@@ -59,21 +56,15 @@ data MempoolParams m where
 data InvReqDataParams key contents m = InvReqDataParams
     { contentsToKey :: contents -> m key
       -- ^ Get key for given contents.
-    , handleInv     :: key -> m Bool
+    , handleInv     :: Maybe NodeId -> key -> m Bool
```

Comment:
New parameter is the provenance of the message.

It should be `NodeId` rather than `Maybe NodeId`. Will fix it.